### ISSUE TITLE
"Enhancement" - LMS App show different shade between pages and assignments in Module section.

### ISSUE THREAD

**Issue by mdhuse** [operepo/ope#113]
When looking at the Modules section on the LMS app the shading for Pages and Assignments are the same. It would be easier to see which one is a page and with one is an assignment if they were shaded differently.

**Comment by mdhuse** [comment]
Yes on the clarification, Good Idea for adding the [P] or [A]. But just thinking that instructors will not want to go through all their courses to make that change of adding a [P] or [A].

**Comment by jmichel-sbctc** [comment]
From the module screen you can edit the name by clicking on the triple dots and then saying edit. Better than opening each document individually at least.. But yeah, I can see it being a pain to go back an do on each assignment\page in each course.
